docs/src/pages/migrations.md

```diff
@@ -17,6 +17,7 @@ This will be the next version of Woodpecker.
   - `CI_PIPELINE_FINISHED` as empty during execution
   - `CI_PIPELINE_STATUS` was always `success`
   - `CI_STEP_STATUS` was always `success`
+- `CI_COMMIT_BRANCH` now always returns the source branch, even for pull requests (previously it returned the target branch)
```

Comment:
As discussed in the chat yesterday. Please add more context. Why, impact and what do I have to change as a user.

Reply:
I think this is the wrong place. This is (currently) a list which briefly lists all migrations without going into much detail.

All points listed there should be explained in more detail in the release notes or in a different place.
Or, if this should be the place, then the whole section should be rewritten in one take rather than doing it for one item here.

Reply:
Ok, but IMO every breaking PRs should already contain the required details notes. Otherwise, someone will have to do a lot of work in release preparation.

Reply:
Why does it not help? At least for this change, we still need it for the v3.0 release or not?

Reply:
Yes, but it would not fit with all the other bullet-list only descriptions that exist right now. 

I committed to do the release notes for 3.0 and hence I have to do all the translation work anyhow. 

It doesn't matter for the *very last* issue of 3.0 now; i.e. forcing a proper migration description within the PR won't change a thing for anyone at this stage.

I don't think this is worth a huge discussion now given all the circumstances 🙃️

Reply:
> Yes, but it would not fit with all the other bullet-list only descriptions that exist right now.

That's why I proposed to create a folder and place a per-pr md file there.

> It doesn't matter for the very last issue of 3.0 now; i.e. forcing a proper migration description within the PR won't change a thing for anyone at this stage.

Well, then we will never change it, and we can skip such governance decisions in the future 🤷

Reply:
I still strongly believe this shouldn't be part of this PR given all the *current* standings but I am repeating myself.

> Well, then we will never change it, and we can skip such governance discussions in the future 

This is a completely different point. I am glad you raised it in the first place. Still I think this is/should be a different discussion/PR. This PR just adopted the scheme for all PRs so far that included some migration notes. I don't think we need a general discussion about a workflow in the (hopefully) very last instance of such a PR. 

> That's why I proposed to create a folder and place a per-pr md file there.

Again, out-of-scope for *this* PR IMO but agreed on.